index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
+                                    <li>Zero, one, or more <code>rdfs:seeAlso</code> to refer to related <a
+                                            href="#extended-profile-documents">documents extending the profile.</a></li>
+                                </ul>
+                                <div class="note" id="other-information" inlist="" rel="schema:hasPart"
+                                    resource="#other-information">
+                                    <h5 property="schema:name"><span>Note</span>:</h5>
+                                    <div datatype="rdf:HTML" property="schema:description">
+                                        <p>The Solid WebID Profiles may include other information pertaining to scenarios in
+                                            which profiles are used or shared as mentioned in <a href="#use-cases">use
+                                                cases</a>.</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+        
+                    <h3 property="schema:name" content="Preferences Document">Preferences Document</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p>The <a href="#private-profile">Preferences Document</a> has the following properties:</p>
+                        <ul>
+                            <li>One <code>rdf:type</code> property whose object is <code>space:ConfigurationFile</code>.</li>
+                            <li>Zero, one, or more <code>rdfs:seeAlso</code>.</li>
+                        </ul>
+                    </div>
+                    <h3 property="schema:name" content="Extended Profile Documents">Extended Profile Documents</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p>The <a href="#extended-profile-documents">Extended Profile Documents</a> are also self-describing
+                            Solid WebID Profiles and share the same data model. <a href="reader-application">Reader
+                                Applications</a> retrieve representations of Extended Profile Documents the same way as they
+                            retrieve Solid WebID Profiles.</p>
+                    </div>
+                    <p about="" id="reader-application-extended-profile" rel="spec:requirement"
+                        resource="reader-application-extended-profile">
+                        <span property="spec:statement"><a rel="spec:requirementSubject" href="#ReaderApplication">Reader
+                                Application</a> <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> retrieve a
+                            representation of an <a href="#extended-profile-documents">Extended Profile Document</a>.</span>
+                    </p>
```

Comment:
As it stands, this is not a testable requirement as there is nothing to say what mechanism would be used.

Reply:
As a MAY, it would typically not be tested, so it doesn't matter if it's testable. That said, the mechanism would typically be dictated by the URL of the Extended Profile Document, and any representation returned by the server would be acceptable.

Reply:
> here is nothing to say what mechanism would be used.

Just before that requirement, the text says:

> Reader Applications retrieve representations of Extended Profile Documents the same way as they retrieve Solid WebID Profiles.

Is this enough to clarify the mechanism?

Reply:
The text before the requirement seems to serve this purpose.